src/toxcore/tcp/server/server_ext.rs

```diff
@@ -285,7 +285,6 @@ mod tests {
     use failure::Error;
 
     use tokio;
```

Comment:
shall we `use tokio`?

Reply:
Indeed, it is useless to import a module path.